modules/lib/lib-grid/src/main/resources/lib/xp/grid.js

```diff
@@ -0,0 +1,35 @@
+const bean = __.newBean('com.enonic.xp.lib.grid.MemoryGridHelper');
+
+function checkRequired(params, name) {
+    if (params[name] === undefined) {
+        throw `Parameter "${name}" is required`;
+    }
+}
+
+exports.get = function (params) {
+    checkRequired(params, 'mapId');
```

Comment:
It would be more convenient to get a map by `mapId` just once and work with it implicitly (similar to how `RepoConnection` in node api works)